### PR TITLE
Added `/apps debug kv` command

### PR DESCRIPTION
**Pull request by levb**
`debug kv [ info | list | edit | clean ]`

### CODE REVIEW

File: server/builtin/debug_kv_edit.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+func (a *builtinApp) debugKVEdit() handler {
+	return handler{
+		requireSysadmin: true,
+
+		commandBinding: func() apps.Binding {
+			return apps.Binding{
+				Label:       "edit",
+				Location:    "edit",
+				Hint:        "",
+				Description: "View or edit specific KV keys of an app.",
```
[the remark this answers is not in the document]
Comment (hanzei) [reply server/builtin/debug_kv_edit.go]: Sure. Would you be open on creating one?

File: server/builtin/debug_kv_edit_modal.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+						ModalLabel: "Action to take",
+						Type:       apps.FieldTypeStaticSelect,
+						SelectStaticOptions: []apps.SelectOption{
+							{
+								Label: "Store New Value",
+								Value: "store",
+							},
+							{
+								Label: "Delete Key",
+								Value: "delete",
```
[the remark this answers is not in the document]
Comment (hanzei) [reply server/builtin/debug_kv_edit_modal.go:59]: Deleting a value that is empty should be seen as confusing. I was wondering of the `delete` button should not be shown if the value is empty. This is non-blocking.

**Review by hanzei — APPROVED** [review]
LGTM besides the changes from https://github.com/mattermost/mattermost-plugin-apps/pull/282 that crept in.

File: server/builtin/debug_kv_edit.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (levb) [reply server/builtin/debug_kv_edit.go]: What is the process, just create a github issue, or do we need to originate in Jira? We can also use an `/apps debug kv create`

**Pull request merged**